pkg/common/common.go

```diff
@@ -0,0 +1,119 @@
+package common
+
+import (
+	"regexp"
+
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+
+	// RoleBindingNames holds the customer rolebindings
+	RoleBindingNames = []string{
```

Comment:
if you did `RoleBindingNames = map[string]struct{}{"customer-admin": {}, ...}` you'd be able to iterate it and do a O(1) lookup - ie not need rxCustomerRoleBinding or CustomerRoleBinding().